synapse/tests/test_lib_scrape.py

```diff
@@ -468,6 +471,44 @@
 dc:\\foo\\bar
 '''
 
+good_uncs = [
+    '\\\\foo\\bar\\baz',
+    '\\\\server\\share',
+    '\\\\server.domain.com\\share\\path\\to\\filename.txt',
+    '\\\\1.2.3.4\\share',
+    '\\\\1.2.3.4\\share\\dirn',
+    '\\\\1234-2345--3456.ipv6-literal.net\\share',
+    '\\\\1234-2345--3456.ipv6-literal.net\\share\\dirn',
+    '\\\\1-2-3-4-5-6-7-8.ipv6-literal.net\\share\\filename.txt',
+    '\\\\server@SSL\\share\\foo.txt',
+    '\\\\server@1234\\share\\foo.txt',
+    '\\\\server@SSL@1234\\share\\foo.txt',
+    '\\\\0--1.ipv6-literal.net@SSL@1234\\share\\foo.txt',
+    ('\\\\server\\share\\' + ('A' * 250) + '.txt:sname:stype\n'),
```

Comment:
tuple?

Reply:
Not a tuple, just building a long string.